spec/controllers/home_controller_spec.rb

```diff
@@ -23,20 +23,41 @@
 
     it '降順選択の場合、新着順であること' do
       get :search, params: { q: { sorts: 'id desc' } }
-      expect(controller.instance_variable_get("@pets")[0]).to eq other_pet
-      expect(controller.instance_variable_get("@pets")[1]).to eq pet
+      expect(controller.instance_variable_get('@pets')[0]).to eq other_pet
+      expect(controller.instance_variable_get('@pets')[1]).to eq pet
```

Comment:
以下のように書くと1行で収まるかなと思いました。

```suggestion
      expect(controller.instance_variable_get('@pets')).to eq [other_pet, pet]
```